package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
@guybedford Did you try both branches?  From your description, it looks like you didn't try the master branch (which is where the double interop issue is) since you mentioned an override.  The master branch has no overrides.  You might also want to try the 8 steps I listed for creating the master branch.